getsolus/packages
# LDC 1.39.0 & Rebuilds

**Summary**

[Changelog](https://github.com/ldc-developers/ldc/releases/tag/v1.39.0)

**Test Plan**

- Use the toolchain to build projects
- Verify tilix stills works

**Checklist**
